great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -201,6 +202,7 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
             runtime_environment (dict): a dictionary of config variables that
                 override those set in config_variables.yml and the environment
         """
+        self.__xdatasource_keys: Set[str] = set()
```

Comment:
Do we want to more widely use `__` prefixes? I know they're different than `_`

Reply:
Works, for me. I'll change it to a `sunder` method 😄

Reply:
Got rid of the whole thing.